_rules/document-title-is-descriptive.md

```diff
@@ -0,0 +1,198 @@
+---
+name: Document title is descriptive
+rule_type: atomic
+description: |
+  This rule checks that the document title describes the topic or purpose of the document.
+
+success_criterion: 
+- 2.4.2 # Page Titled (A)
+
+test_aspects:
+- DOM Tree
+
+authors:
+- Anne Thyme Nørregaard
+- Corbb O’Connor 
+
+---
+
+## Test procedure
+
+### Applicability
+
+This rule applies to the first HTML `title` element in a [document](https://www.w3.org/TR/dom/#concept-document) where the [document element](https://www.w3.org/TR/dom/#document-element) is an HTML `<html>` element, if the first HTML `title` element is [non-empty](#non-empty).
+
+### Expectation
+
+Each target element describes the topic or purpose of the overall content of the [document](https://www.w3.org/TR/dom/#concept-document).
```

Comment:
There is only one target element so we shouldn't use "Each". "The target element" seems more appropriate.